editor/src/components/editor/actions/actions.tsx

```diff
@@ -1517,7 +1517,11 @@ function createStoryboardFileIfMainComponentPresent(
 
 function createStoryboardFileWithPlaceholderContents(
   projectContents: ProjectContentTreeRoot,
+  isItNecessary: 'create-placeholder' | 'skip-creating-placeholder',
```

Comment:
Be nice to extract this into a type definition as it appears to be in at least 2 places.

Reply:
Also `isItNecessary` is a bad name. `shouldCreatePlaceholder` or just `createPlaceholder` would be better

Reply:
I did the renaming, I did not create a type definition though because I think this is more readable. we should talk about it in the chat if you disagree :)